example/src/hooks/useThemeActions.tsx

```diff
@@ -31,6 +31,7 @@ export default function useThemeActions() {
       name: "Change theme…",
       keywords: "interface color dark light",
       section: "Preferences",
+      shortcut: ["$mod+1"]
```

Comment:
Hitting mod+1 will trigger a browser tab switch as this event default is not prevented.